exercises/all-your-base/examples/src/AllYourBase.purs

```diff
@@ -0,0 +1,24 @@
+module AllYourBase
+  ( rebase
+  ) where
+
+import Prelude
+import Data.Array (any, head, length, range, reverse, zipWith, (:))
+import Data.Foldable (sum)
+import Data.Int (pow)
+import Data.Maybe (Maybe(..), maybe)
+
+rebase' :: Int -> Int -> Array Int -> Array Int
+rebase' inb oub ind = toDec # fromDec # reverse
+  where toDec = zipWith (*) ind pows # sum
```

Comment:
Sure thing, updated. Interesting, the original form seems more natural to me. :)